crates/red_knot_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -627,6 +692,21 @@ impl PackageKind {
     }
 }
 
+pub(super) struct ResolverContext<'db> {
+    pub(crate) db: &'db dyn Db,
+    pub(crate) target_version: PythonVersion,
```

Comment:
These probably don't need to be `pub(crate)` anymore now that the whole module resolver is a submodule of `red_knot_python_semantic` (probably my oversight in the earlier PR)

```suggestion
    pub(super) db: &'db dyn Db,
    pub(super) target_version: PythonVersion,
```